mobile/src/people/avatar.tsx

```diff
@@ -1,49 +1,48 @@
 import React, { Component } from 'react';
-import { View, Image, StyleSheet, ViewStyle, ImageStyle } from 'react-native';
+import { View, Image, StyleSheet, ViewStyle, ImageStyle, LayoutChangeEvent } from 'react-native';
 
 const styles = StyleSheet.create({
     outerFrame: {
         justifyContent: 'center', 
         alignItems: 'center',
-        // Following attributes are default and could be changed via component props
-        height: 156,
-        width: 156,
-        borderRadius: 156 * 0.5,
+        // Following attributes are default
         borderColor: '#2FAFCC',
-        borderWidth: 2
+        borderWidth: 2,
+        height: 200,
+        width: 200,
+        borderRadius: 200 * 0.5,
     },
     image:  {
-        // Following attributes are default and could be changed via component props
-        height: 154,
-        width: 154,
-        borderRadius: 154 * 0.5,
+        // Following attributes are default
         borderColor: '#fff',
-        borderWidth: 2
+        borderWidth: 2,
     }
 });
 
 export interface AvatarProps {
     mimeType: string;
     photoBase64: string;
-    // These attributes should be passed in following style props:
-    // height, width, borderRadius, borderColor, borderWidth
-    //
-    // Border radius is used for circle mask effect and should be 50% of width/height
-    //
-    // Image border color could be same as superview's background color 
-    // or some other color - refer to design.
-    outerFrameStyle: ViewStyle;
-    imageStyle: ImageStyle;
+    style: ViewStyle;
 }
 
 export class Avatar extends Component<AvatarProps> {
+    private borderRadius: number;
+    
+    public onLayout = (e: LayoutChangeEvent) => {
+        this.borderRadius = e.nativeEvent.layout.width * 0.5;
```

Comment:
any reason why `this.borderRadius` is a field instead of a parameter in 'setState'?

Reply:
it actually depends on call order whether you'll get a valid radius in render() method or not... https://reactjs.org/docs/faq-state.html